packages/web-app-files/src/services/folder/loaderPublicFiles.ts

```diff
@@ -5,9 +5,9 @@ import { DavProperties, DavProperty } from 'web-pkg/src/constants'
 import { buildResource } from '../../helpers/resources'
 import { isLocationPublicActive, createLocationPublic } from '../../router'
 
-import { linkRoleUploaderFolder } from '../../helpers/share'
 import { Store } from 'vuex'
 import { authService } from 'web-runtime/src/services/auth'
+import { linkRoleUploaderFolder } from 'web-client/src/helpers'
```

Comment:
You're sometimes importing from `web-client/src/helpers` and sometimes from `web-client/src/helpers/share`. IMO the latter is more intuitive. So you'd need to drop the export statement in `web-client/src/helpers/index.ts`.

Reply:
🤷🏻‍♂️